unified-codes/libs/ui/src/lib/components/inputs/molecules/index.ts

```diff
@@ -1,10 +1,6 @@
-export * from './ClearButton';
 export * from './ClearInput';
 export * from './IconInput';
 export * from './LoginInput';
 export * from './PasswordInput';
 export * from './SearchBar';
-export * from './SearchButton';
-export * from './ToggleButton'
 export * from './UsernameInput';
-export * from './ToggleButtonGroup';
```

Comment:
@katherine-sussol I killed some of your components 😢.

I think I may have set a bad precedent with `ClearButton` etc. which influenced `ToggleButton`. Have now realised it's bad practice to create components which implement a specific case of a child's functionality. I think the motivation for doing this is from our background in OOP (start with a generic parent class and inherit more specific cases from it), but with compositional design, I think the idea is to build from specific to general by defining reusable sets of commonly grouped smaller components (and ideally with the smaller components exposed as props!). 

From above, I think `ToggleButton` is like `ClearButton` etc. in terms of defining a component which implements a subset of functionality of a general component (rather than superset of functionality of multiple components), so something to avoid, I think? `ToggleButtonGroup` is more the sort of thing to go with, only reason it's gone is that I found that it's basically covered by `EntityToggleBar` for now (I'm also still not sure whether its useful or not to define components which group one specific component, haven't got a strong opinion either way ATM!)... but I think we could look at a `ButtonGroup` component if it comes up again?